src/coreclr/jit/codegen.h

```diff
@@ -803,7 +803,16 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
     void genSetRegToCond(regNumber dstReg, GenTree* tree);
 
 #if defined(TARGET_ARMARCH) || defined(TARGET_LOONGARCH64) || defined(TARGET_RISCV64)
-    void genScaledAdd(emitAttr attr, regNumber targetReg, regNumber baseReg, regNumber indexReg, int scale);
+    void genScaledAdd(emitAttr  attr,
+                      regNumber targetReg,
+                      regNumber baseReg,
+                      regNumber indexReg,
+                      int       scale
+#ifdef TARGET_RISCV64
+                      ,
+                      regNumber scaleTempReg
```

Comment:
Please introduce something similar to `UNIX_LOONGARCH64_ONLY_ARG`.

Reply:
There's already RISCV64_ARG in stdmacros.h, I think I can reuse.